torch/nn/_functions/thnn/auto_symbolic.py

```diff
@@ -26,6 +26,10 @@ def softmax_symbolic(g, input):
     return g.op('Softmax', input)
 
 
+def logsoftmax_symbolic(g, input):
+    return g.op("Log", g.op('Softmax', input).typeAs(input))
```

Comment:
Doesn’t ONNX have log softmax? That’s surprising. Softmax + Log is very unstable in backward

Reply:
Unfortunately, no logsoftmax in ONNX. For now, I think it's OK, because the exported model only runs inference. If we decide to enable backward, this may cause problem.

Reply:
We should add LogSoftMax (sometimes called LogSumExp) to ONNX I think. It's pretty important for numerical stability, even in forward and most libraries implement it. Create an issue or PR?

Reply:
Done in https://github.com/onnx/onnx/issues/115